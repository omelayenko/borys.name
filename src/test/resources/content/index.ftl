<ul>${topic_blog}</ul>
