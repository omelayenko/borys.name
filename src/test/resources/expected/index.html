<!DOCTYPE html>
<html>
<title>1</title>
<body>
<ul><li>Blog Two: Blog Two Descr</li>
<li>Blog One: Blog One Descr</li></ul>
</body>
</html>